CryptoTechReminderSystem.AcceptanceTest/ApiEndpointResponse/HarvestTimeEntriesEndOfTheMonthResponse.json

```diff
@@ -1,65 +1,90 @@
 {
```

Comment:
Add `project` and `is_closed` attributes to existing `time_entries` test data. (Relevant Harvest [docs](https://help.getharvest.com/api-v2/timesheets-api/timesheets/time-entries/))

Note that `is_closed` refers to a time entry that has been _submitted and approved_.  A time entry that is submitted but not approved is still open.